cmd/posix.go

```diff
@@ -384,11 +384,60 @@ func (s *posix) CrawlAndGetDataUsage(ctx context.Context, cache dataUsageCache)
 			return 0, nil
 		}
 
+		if cache.Info.lifeCycle != nil {
+			objName, err := filepath.Rel(filepath.Join(s.diskPath, cache.Info.Name), item.Path)
```

Comment:
we never really deal with windows specific paths AFAIK the paths returned are always with `/` as separator, this is why we used `path` everywhere instead of filepath. 

If needed why not we convert `filepath.ToSlash()` and then use `path.Rel(path.Join())` instead?

Reply:
There is no `path.Rel()`